### ISSUE TITLE
Scrollcontainer's scrollbars aren't initialized properly when opening a scene in the editor

### ISSUE THREAD

**Issue by AaronRecord** [godotengine/godot#48057]
**Godot version:**

3.3.rc9, does **not** occur in 3.2.3

**OS/device including version:**

Windows 10

**Issue description:**

```
# ScrollContainer.tscn
...
scroll_horizontal = 20
scroll_vertical = 30
```

https://user-images.githubusercontent.com/50304111/115482705-8800e780-a20c-11eb-922f-7de24159407a.mp4

I suspect this is regression from #43833 where the first time `update_scrollbars()` is run `min = (0.0, 0.0)`, which sets `scroll_horizontal` and `scroll_vertical` back to `0.0` from their correct values

**Steps to reproduce:**
See above and below

**Minimal reproduction project:**

[minimal_reproduction_project.zip](https://github.com/godotengine/godot/files/6347383/ScrollcontainerMRP.zip)

**Comment by KoBeWi** [comment]
For some reason it does not seem to happen on master (f6d5b2f)

**Comment by LightningAA** [comment]
Found another bug that happens in 3.3 but not 3.2, if you:
- have horizontal scrolling enabled and have h scroll at some non-zero value
- increase the horizontal size until the horizontal scrollbar disappears
- undo
Now the horizontal scrollbar is back at 0 even though h scroll isn't

**Comment by akien-mga** [comment]
Still reproducible in 3.5 beta 5.

**Comment by akien-mga** [comment]
Fixed by #61346.
